modules/end-user-guide/pages/git-credential-store.adoc

```diff
@@ -1,25 +1,79 @@
 
 .Prerequisites
 
 * A running instance of {prod-short}.
+* You have CLI or GUI access to the {orch-name} cluster of your organization's {prod-short} instance.
+
+ifeval::["{project-context}" == "che"]
+** For CLI users: `{orch-cli}` is installed in the operating system you are using. See link:https://kubernetes.io/docs/tasks/tools/#kubectl[Install Tools: kubectl].
+endif::[]
+
+ifeval::["{project-context}" == "crw"]
+** For CLI users: `{orch-cli}` is installed in the operating system you are using. See link:https://docs.openshift.com/container-platform/4.10/cli_reference/openshift_cli/getting-started-cli.html#installing-openshift-cli[Installing the OpenShift CLI].
+endif::[]
+
+* For CLI users: link:https://www.gnu.org/software/coreutils/base64[`base64`] is installed in the operating system you are using.
 
 .Procedure
-. Prepare Git credential file in the link:https://git-scm.com/docs/git-credential-store#_storage_format[Storage format].
-. Encode content of the file to the base64 format.
-. Create a new {platforms-name} secret in the {platforms-name} {orch-namespace} where a {prod-short} workspace will be created.
+. In your home directory, locate and open your `.git-credentials` file if you already have it. Alternatively, if you do not have this file, then save a new `.git-credentials` file, using the following link:https://git-scm.com/docs/git-credential-store#_storage_format[Git credentials storage format]. Each credential is stored on its own line in the file:
+
++
+[subs="+quotes,+attributes,+macros"]
+----
+https://__<username>__:__<token>__@__<git_server_hostname>__
+----
+
++
+.A line in a `.git-credentials` file
+====
+`https://trailblazer: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@github.com`
+====
 
-* The labels of the secret that is about to be created must match the set of labels configured in `che.workspace.provision.secret.labels` property of {prod-short}. The default labels are:
+. Select credentials from your `.git-credentials` file to provide to the Secret. Encode the selected credentials to Base64 for the next step.
 
-* `app.kubernetes.io/part-of: che.eclipse.org` 
-* `app.kubernetes.io/component: workspace-secret`:
+. Create a new {platforms-name} secret in the user's {orch-namespace}.
 +
-include::example$snip_{project-context}-git-credential-file-as-kubernetes-secret-into-the-file.adoc[leveloffset=+1]
+[source,yaml,subs="+quotes,+attributes,+macros"]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: git-credentials-secret
+  labels:
+    controller.devfile.io/git-credential: 'true' <1>
+    controller.devfile.io/watch-secret: 'true'
+data:
+  credentials: __<Base64_content_of_.git-credentials>__ <2>
+----
++
+<1> The `controller.devfile.io/git-credential` label marks the secret as containing Git credentials. All Git credential secrets in the same {orch-namespace} will be merged into a single secret that will be mounted to the {devworkspace} containers. All the original secrets are kept intact.
```

Comment:
If the following makes sense, it would clarify things a bit to the reader:
```suggestion
<1> The `controller.devfile.io/git-credential` label marks the Secret as containing Git credentials. All Git credential Secrets in your user {orch-namespace} will be merged into a single Secret that will be mounted to the {devworkspace} containers. All of your original Secrets remain intact.
```